bot/codeSharingBot.ts

```diff
@@ -50,10 +57,15 @@ async function createCardCommand(
   // If URL contains `github`, use GitHub API route.
   if (url.includes("github.com")) {
     return await handleGitHubUrl(url);
+  } else if (url.includes(".visualstudio.com")) {
```

Comment:
It looks the same code logic in handleTeamsAppBasedLinkQuery(), and it is better to refactor it later for long-term code maintenance.

Reply:
Refactored. Thanks~